src/configuration/locales/fr/translations.json

```diff
@@ -1,3 +1,33 @@
 {
-  "Take the survey!": "Réponder au questionnaire!"
+  "...Loading": "...Chargement",
+  "Beekeeper": "Apiculteur",
+  "Beginner": "Débutant",
+  "Billed: {{value}}": "Payant: {{value}}",
+  "Description {{description}}:": "Description: {{description}}",
+  "Experience: {{experience}}": "Expérience: {{experience}}",
+  "Expert": "Expert",
+  "FreeSpace": "Espace disponible",
+  "Garden": "Jardin",
+  "HAPPYCULTEUR © COPYRIGHT 2019. ALL RIGHTS RESERVED.": "Happyculteur © COPYRIGHT 2019. TOUS DROITS RÉSERVÉS.",
```

Comment:
The translation should be in lower case like `Happyculteur © Copyrights 2019. Tous droits réservés.`
Or maybe "some/certains" because the code is Open Source?

Reply:
I totally dunno about the legislations regarding this

Reply:
Any idea @RiceBowlJr